landlab/components/groundwater/dupuit_percolator.py

```diff
@@ -334,6 +344,7 @@ def __init__(
 
         # save courant_coefficient (and test)
         self._courant_coefficient = courant_coefficient
+        self._vn_coefficient = vn_coefficient
```

Comment:
Very minor comment, and something that I recently learned... 

If you do 


```python
self.courant_coefficient = courant_coefficient
self.vn_coefficient = vn_coefficient
```
(removing the leading underscores) then you invoke the setters and use the checking you've built into them.

Reply:
Interesting, should I do that? And if I do, do I remove leading underscores on all appearances of those variables?

Reply:
I would recommend you do. 

When you do this, the assignment on these lines will invoke the setter, which will check the value provided and then assign the value to the private variable`self._courant...` (because that is what the setter does. It is a way to ensure that the checking created in the setter is used in the init. 

Regarding whether there is a difference between using `self._name` of var or `self.name_of_var` (which points to `self._name_of_var `through the `@property`) internally... this is beyond my python knowledge. I might expect that using `self.name_of_var` might be marginally slower. tagging @mcflugen if he has any thoughts that we can both learn from.

Reply:
ok, made some changes to use setters in more places in init. Let me know what you think.

Reply:
looks good to me.